plaso/cli/helpers/elastic_ts_output.py

```diff
@@ -0,0 +1,172 @@
+# -*- coding: utf-8 -*-
+"""The Elastic Timesketch output module CLI arguments helper."""
+
+import getpass
+import json
+import os
+
+from uuid import uuid4
+
+from plaso.cli.helpers import interface
+from plaso.cli.helpers import manager
+from plaso.cli.helpers import server_config
+from plaso.cli import logger
+from plaso.lib import errors
+from plaso.output import shared_elastic
+
+
+class ElasticTimesketchArgumentsHelper(server_config.ServerArgumentsHelper):
+  """Elastic Timesketch CLI arguments helper."""
+
+  _DEFAULT_SERVER = '127.0.0.1'
+  _DEFAULT_PORT = 9200
+
+
+class ElasticTimesketchOutputArgumentsHelper(interface.ArgumentsHelper):
+  """Elastic Timesketch output module CLI arguments helper."""
```

Comment:
this helper is likely not needed since elastic already supports SharedElasticsearchOutputModule https://github.com/log2timeline/plaso/blob/master/plaso/cli/helpers/elastic_output.py#L123

I opt to extend TimesketchOutputArgumentsHelper with ElasticTimesketchOutputModule support https://github.com/log2timeline/plaso/blob/master/plaso/cli/helpers/timesketch_output.py#L76

Reply:
yes, it's needed, we need to set the `timeline_id`, changed this